js/ui/windowManager.js

```diff
@@ -1379,6 +1391,46 @@ const WindowManager = new Lang.Class({
                                onOverwriteParams: [shellwm, actor]
                              });
         }
+
+        this._codingAddAppWindow(actor);
+    },
+
+    _codingAddAppWindow : function(actor) {
```

Comment:
There is a mixed style of "space before colon" vs no space in the file. I sticked now to remove the space for my parts.